core-rust/core-api-server/src/core_api/conversions/addressing.rs

```diff
@@ -140,6 +140,10 @@ pub fn to_api_substate_id(
             SubstateType::BootLoaderModuleFieldKernelBoot,
             models::PartitionKind::Field,
         ),
+        TypedSubstateKey::BootLoader(TypedBootLoaderSubstateKey::BootLoaderField(
+            BootLoaderField::TransactionValidationConfiguration,
+        )) => todo!(),
```

Comment:
Just to confirm that we've got a good way to track all these todos. :)

Reply:
Two parts:
* A work item to Ctrl+F for all the todo!s
* And API tests which will fail when Cuttlefish is enabled and we try sending in transaction v2s.